src/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css"
+  />
+  <title>Food Network</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar container d-flex">
+        <div class="logo">
+          <a href="index.html"><h1>Foodipy</h1></a>
+        </div>
+        <ul class="nav-menu d-flex">
+          <li class="nav-item"><a href="index.html">View more</a></li>
+          <li class="nav-item"><a href="index.html"><span class="item-count">0</span> Meal(s) found!</a></li>
```

Comment:
- [ ] Based on project requirements, the page should have a navigation similar to the given wireframe. Pay attention that the wireframe has 3 pages, Spaces, Planets, Races:
<img width="541" alt="Screenshot 2022-03-10 at 18 33 49" src="https://user-images.githubusercontent.com/77038610/157683763-294cf8fe-f8d0-4d5b-98ac-8b05ad637d8a.png">

Please add 3 pages based on some category, for example, you can make deserts, vegetarian, seafood, like I did in my capstone project, feel free to check it [here](https://shoirata.github.io/RecipeApp/)

Reply:
hi @ShoiraTa, I thought the wireframe was there as a guide, not a requirement. the view more on the navigation navigates between more than just 3 categories, when you click on view more or any of the navigations it randomly selects a new category. Thanks for the review

Reply:
Thanks @ShoiraTa